kodilla-collections/src/main/java/com/kodilla/collections/arrays/homework/CarsApplication.java

```diff
@@ -0,0 +1,41 @@
+package com.kodilla.collections.arrays.homework;
+
+import com.kodilla.collections.interfaces.homework.Audi;
+import com.kodilla.collections.interfaces.homework.BMW;
+import com.kodilla.collections.interfaces.homework.Car;
+import com.kodilla.collections.interfaces.homework.Mercedes;
+
+import java.util.Random;
+
+public class CarsApplication {
+
+    private static final Random random = new Random();
+
+    public static void main(String[] args) {
+
+        Car[] cars = new Car[random.nextInt(15) + 1];
+        for (int n = 0; n < cars.length; n++)
+            cars[n] = drawCar();
+        for (Car car : cars)
+            CarUtils.describeCar(car);
+    }
+
+    public static Car drawCar() {
+
+        int increaseSpeed = 50;
+        int drawnCars = random.nextInt(3);
+        if (drawnCars == 0)
+            return new Mercedes(increaseSpeed,0);
+        else if (drawnCars == 1)
+            return new BMW(increaseSpeed, 0);
+        else {
+            return new Audi(increaseSpeed, 0);
+        }
+    }
+
+    public static int getRandomIncreaseSpeed(Car car) {
+        for (int i = 0; i < random.nextInt(5) + 1; i++)
+            car.increaseSpeed();
+        return ////;
+    }
```

Comment:
Dalczego interface Car oraz jego implementacje nie znalazly sie w pull requescie?
Klasy implementujace interfejs Car powinny miec tylko jeden (bezparametrowy) konstruktor. 
Przyklad:
public class Mercedes implements Car {
    private int speed;
    public Mercedes() {
      speed = 0;  // nowo utworzony samochod raczej nie ma predkosci. Ewentualnie mozesz potem dodac drugi konstruktor (jesli bedzie potrzeba Mercedes(int speed) { this.speed = speed; } )
    }

    @Override
    public int getSpeed() {
        return speed;
    }

    @Override
    public void increaseSpeed() {
        speed += increaseSpeed;
    }

    @Override
    public void decreaseSpeed() {
        speed -= decreaseSpeed;
       if (speed < ) {
          speed = 0;    // predkosc samochodu raczej nie jest wartoscia ujemna, wiec nie mozna np majac predkosc samochodu 50 km/h zmniejszyc jej o 100 km/h, tylko wyzerowac)
       }
    }
}

Reply:
interface Car oraz jego implementacje były w poprzednim zadaniu i pull requescie. Zostało zatwierdzone dlatego nic tam nie ruszałem już.